action.yml

```diff
@@ -131,5 +131,5 @@ runs:
           }
         }
         Write-Output "Version=$Version"
-        Write-Output "::set-output name=version::$Version"
+        $GITHUB_OUTPUT += "version = $Version"
```

Comment:
Hey this is almost right,

```powershell
"bin=${etc}" >> $env:GITHUB_OUTPUT
```

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions?tool=powershell#setting-an-output-parameter

Reply:
Fixing :-)

Reply:
Fixed